src/Microsoft.Diagnostics.Monitoring.WebApi/Models/SpeedScopeStackResults.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Models
+{
+    public class SpeedscopeResult
+    {
+        [JsonPropertyName("exporter")]
+        public string Exporter { get; set; }
+
+        [JsonPropertyName("name")]
+        public string Name { get; set; }
+
+        [JsonPropertyName("activeProfileIndex")]
+        public int ActiveProfileIndex { get; set; }
+
+        [JsonPropertyName("$schema")]
+        public string Schema { get; set; } = "https://www.speedscope.app/file-format-schema.json";
+
+        [JsonPropertyName("profiles")]
+        public List<Profile> Profiles { get; set; }
+
+        [JsonPropertyName("shared")]
+        public SharedFrames Shared { get; set; }
+    }
+
+    public class SharedFrames
+    {
+        [JsonPropertyName("frames")]
+        public List<SharedFrame> Frames { get; set; }
+    }
+
+    public class SharedFrame
+    {
+        [JsonPropertyName("name")]
+        public string Name { get; set; }
+
+        [JsonPropertyName("col")]
+        public int? Col { get; set; }
```

Comment:
nit: Let's avoid abbreviations in the property names:

```suggestion
        public int? Column { get; set; }
```